src/main/java/frc/robot/commands/IntakeCommand.java

```diff
@@ -31,7 +34,13 @@ public void initialize() {}
   // Called every time the scheduler runs while the command is scheduled.
   @Override
   public void execute() {
-    intakeSubsystem.setIntakeSpeed(Constants.IntakeConstants.INTAKE_PERCENTAGE);
+    if (pos.equals("TURRET")) {
+      intakeSubsystem.setIntakeSpeed(Constants.IntakeConstants.INTAKE_PERCENTAGE);
+    } else if (pos.equals("TRAP")) {
+      intakeSubsystem.reverseChooser(Constants.IntakeConstants.INTAKE_PERCENTAGE);
```

Comment:
The intakeSpeed still needs to be run when sending to the trap

Let's break this into two commands, where one runs the chooser and one runs the intake.  
We will have the chooser wheel get where it is going from the upcoming robot state management system